IPS.MVC/Controllers/HomeController.cs

```diff
@@ -0,0 +1,37 @@
+﻿using IPS.MVC.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace IPS.MVC.Controllers
+{
+    public class HomeController : Controller
+    {
+        private readonly ILogger<HomeController> _logger;
+
+        public HomeController(ILogger<HomeController> logger)
```

Comment:
```suggestion
        public HomeController(ILogger<HomeController> logger)
        {
            _logger = logger;
        }
```

可以直接  => _logger = logger;

Reply:
考虑到有一个需要教小白的问题（昨天fork代码的两个），所以前期不打算使用lamda表达式，等有个一两天他们会了的时候再使用。